packages/vaadin-list-box/src/vaadin-list-box.d.ts

```diff
@@ -17,7 +17,7 @@ export type  ListBoxSelectedChanged = CustomEvent<{ value: number }>;
 /**
  * Fired when the `selectedValues` property changes.
  */
-export type  ListBoxSelectedValuesChanged = CustomEvent<{ value: Array<string> }>;
```

Comment:
Please also change the test accordingly:

https://github.com/vaadin/vaadin-web-components/blob/e5c6645303b5473bd909fe4057bb99277b89bfea/packages/vaadin-list-box/test/typings/list-box.types.ts#L15-L17

Reply:
Noticed, updated!